core/api/models.py

```
@@ -1,3 +1,81 @@
 from django.db import models
+from uuid import uuid4
+from auth_app.models import CustomUser
+
+
+
+class contribution_videos(models.Model):
```

Ruff (0.8.2)

Model does not define `__str__` method

(DJ008)